client/src/components/MapPin/MapPin.js

```diff
@@ -0,0 +1,28 @@
+//Heavily influenced by example code here:
+//https://github.com/uber/react-map-gl/blob/5.2-release/examples/controls/src/pins.js
+
+import React, {PureComponent} from 'react';
+import {Marker} from 'react-map-gl';
+import { ReactComponent as Pin } from '../../images/map-marker-icon.svg';
+
+import './MapPin.css';
+
+export default class MapPin extends PureComponent {
+  render() {
+    const {data, onClick} = this.props;
+
+    return data.map((facility, index) => (
+      <Marker key={`marker-${index}`} longitude={facility.longitude} latitude={facility.latitude}>
+        <svg className="MapPin-Icon"
+             id={this.facility_id}
+             onClick={() => this.props.onClick(facility.location_id)}
```

Comment:
`this.props.onClick()` could be `this.onClick()` because the line 12 already deconstructed the `onClick`

Reply:
oh, yeah good point. I was just modifying example code on this one and missed that was there.